yamllint/rules/comments.py

```diff
@@ -21,6 +21,9 @@
 
 * Use ``require-starting-space`` to require a space character right after the
   ``#``. Set to ``true`` to enable, ``false`` to disable.
+* Use ``ignore-shebang`` to ignore a shebang at the beginning of the file.
+  (https://en.wikipedia.org/wiki/Shebang_(Unix)) when checking
+  ``require-starting-space``
```

Comment:
There's an extra `.` on line 24, and a `.` is missing on line 26. Also, are you sure the URL is going to be correctly rendered in RST? (You can use `python3 setup.py build_sphinx` to generate HTML docs.)

Reply:
No idea about RST :) I'll read up on it!